kwave/utils/conversion.py

```diff
@@ -366,7 +391,13 @@ def hounsfield2density(ct_data: np.ndarray, plot_fitting: bool = False) -> np.nd
 subs0 = None
 
 
-def tol_star(tolerance, kgrid, point, debug):
+@beartype
+def tol_star(
+    tolerance: NUMERIC, 
+    kgrid: kWaveGrid, 
+    point: NDArray, 
```

Comment:
only 1D 2D or 3D?

Reply:
1D with up to 3 elements, updated the annotation